app/controllers/teachers/guides_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Teachers
+  class GuidesController < BaseController
+    def getting_started
+      @guide_embed_code = '<iframe src="https://docs.google.com/document/d/11sP1Y1KM6Xh5vmjsuJSmvfveT6MZORlwxNG2ErTMwmw/pub?embedded=true
```

Comment:
How about we just put the iframe directly in the HTML? That way we just have all the HTML in one place instead of having some in the controller.